src/Nethermind/Nethermind.Optimism/OPConfigHelper.cs

```diff
@@ -6,16 +6,16 @@
 
 namespace Nethermind.Optimism;
 
-public class OptimismSpecHelper(OptimismParameters parameters) : IOptimismSpecHelper
+public class OptimismSpecHelper(OptimismChainSpecEngineParameters parameters) : IOptimismSpecHelper
 {
-    private readonly long _bedrockBlockNumber = parameters.BedrockBlockNumber;
-    private readonly ulong _regolithTimestamp = parameters.RegolithTimestamp;
+    private readonly long? _bedrockBlockNumber = parameters.BedrockBlockNumber;
+    private readonly ulong? _regolithTimestamp = parameters.RegolithTimestamp;
```

Comment:
Why do we have to make stuff nullable?

Reply:
There might be no value in config, so better to make it nullable imo

Reply:
@deffrian shouldn't we always have a value for these hardforks?